### PR TITLE
Implement desktop styling based on Figma template design

### PR DESCRIPTION
**Pull request by ahmidou09**
This pull request includes the implementation of desktop styling for the project.
The goal is to replicate the design as closely as possible, including the layout, fonts, colors, images, text, and margins specified in the Figma template.
Media queries have been used in the CSS file to ensure that the desktop version matches the selected template (Desktop: >=768px).
The layout elements have been organized using Grid and Flexbox for optimal arrangement.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,164 +87,176 @@ <h1 class="heading-1">I’m Alexandra. Glad to see you!</h1>
...
-              <li>JavaScript</li>
-            </ul>
+              <ul class="tags">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
 
-            <button type="button" class="btn-primary">See project</button>
+              <button type="button" class="btn-primary">See project</button>
```
Comment (tufoinnkuo10) [inline index.html:181]: - [ ] Great job with the media query and making your work responsive. Please kindly reduce the margin left and right so it matches the design. See the difference below. kindly implement this till the contact section.
yours
Screenshot (1110)_LI
figma
Screenshot (1111)

File: index.html [hunk 2]
```diff
@@ -269,58 +301,63 @@ <h2 class="mb-1 heading-1">About Myself</h2>
...
-                ></textarea>
-              </li>
-              <li class="form-item">
-                <button type="submit" class="form-btn">Get in touch</button>
-              </li>
-            </ul>
-          </form>
+          <div class="center">
+            <h2 class="heading-2">Contact me</h2>
+            <p class="form-para">
```
Comment (tufoinnkuo10) [inline index.html:310]: - [ ] Great job. please note that the contact section is full width. kindly fix this to align with the figma design.
yours
Screenshot (1108)_LI
figma
Screenshot (1109)

File: index.html [hunk 3]
```diff
@@ -67,164 +87,176 @@ <h1 class="heading-1">I’m Alexandra. Glad to see you!</h1>
...
-              <li>JavaScript</li>
-            </ul>
+              <ul class="tags">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
 
-            <button type="button" class="btn-primary">See project</button>
+              <button type="button" class="btn-primary">See project</button>
```
Comment (tufoinnkuo10) [inline index.html:115]: - [ ] please kindly ensure the desktop version has the button interaction of hover and pressed like the original design.
Screenshot (1112)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @ahmidou09 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ followed figma design

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 4]
```diff
@@ -234,6 +254,13 @@ body {
   gap: 36px;
 }
 
+.section-about-info {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  gap: 12px;
+}
+
```
Comment (Owusu-Desmond) [inline style.css:263]: - [ OPTIONAL ] Please make sure that the About section matches the Figma design, the list languages should align right on the desktop screen as you can see below :point_down: 

  ### YOUR DESIGN
  image
  
  ### FIGMA DESIGN
  image

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ahmidou09 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (ahmidou09) [reply style.css:263]: ok thank you

**Pull request merged**